pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardfull/_generated_object_monitoringdashboardfull.golden.yaml

```diff
@@ -129,9 +129,8 @@ spec:
               project_id: my-project
               zone: us-central1-a
             type: gce_instance
-          policyNames:
-          - foo
-          - bar
+          policyRefs:
+          - name: monitoringalertpolicy-${uniqueId}
```

Comment:
nit: suggest using two policyRefs to test slice behavior.

Reply:
Also, I think we may want to sort this list to avoid unnecessary requests to GCP. (defer to you, this is minor since the monitoring dashboard uses etag)